examples/memodel/memodel.py

```diff
@@ -0,0 +1,402 @@
+"""Get EModel, modify morphology, plot analysis and upload MEModel"""
+
+# Attention! This will overwrite figures made with the same access point.
+# It is highly recommended to clean the figures folder between two runs
+# to avoid any leftover figure to be wrongly associated with a MEModel.
+
+import copy
+import getpass
+import pathlib
+
+from kgforge.core import KnowledgeGraphForge
+from kgforge.specializations.resources import Dataset
+
+from bluepyemodel.emodel_pipeline.memodel import MEModel
+from bluepyemodel.emodel_pipeline.plotting import plot_models
+from bluepyemodel.emodel_pipeline.plotting import scores
+from bluepyemodel.evaluation.evaluation import compute_responses
+from bluepyemodel.evaluation.evaluation import get_evaluator_from_access_point
+from bluepyemodel.tools.search_pdfs import copy_emodel_pdf_dependencies_to_new_path
+from bluepyemodel.validation.validation import compute_scores
+
+from bluepyemodel.access_point.forge_access_point import get_brain_region_notation
```

Comment:
Could you put those two imports above the other bpem imports? So that they are ordered